addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -717,6 +742,7 @@ def migrate(env, version):
     fill_move_line_invoice(cr)
     merge_invoice_journals(env)
     update_account_invoice_date(cr)
+    update_move_date(cr)
```

Comment:
Shouldn't be this line before than the previous one? :thinking:

Reply:
It doesn't matter

Reply:
Ok